src/elm/Pages/Org_/Repo_/Schedules.elm

```diff
@@ -376,8 +381,34 @@ viewSchedule zone org repo schedule =
         ]
 
 
-{-| addKey : helper to create Vela.Schedule key
--}
-addKey : Vela.Schedule -> Vela.Schedule
-addKey schedule =
-    { schedule | org = schedule.org ++ "/" ++ schedule.repo ++ "/" ++ schedule.name }
+scheduleErrorRow : Vela.Schedule -> Maybe (Components.Table.Row Vela.Schedule msg)
+scheduleErrorRow schedule =
+    if not <| String.isEmpty schedule.error then
+        Just <| Components.Table.Row schedule viewScheduleError
+
+    else
+        Nothing
+
+
+viewScheduleError : Vela.Schedule -> Html msg
+viewScheduleError schedule =
+    let
+        lines =
+            Utils.Ansi.decodeAnsi schedule.error
+                |> Array.map
+                    (\line ->
+                        Just <|
+                            Ansi.Log.viewLine line
+                    )
+                |> Array.toList
+                |> List.filterMap identity
```

Comment:
do we actually need to deal with Ansi here? it would always be straight text, correct?

Reply:
Good point. Yeah, the errors should show as plain text. I copied what was used for `viewHookError` and applied it to `viewScheduleError`. Should I remove `decodeAnsi ` from both? @plyr4 What are your thoughts?

Reply:
hook errors can contain ansi, see: https://github.com/go-vela/ui/pull/305
though, maybe that was added in the small timeframe where we switched over to `goccy/yaml` that was producing pretty errors. i cant remember, it was 3 years ago 😅

Reply:
lol yup the timing is lining up, here's the PR where we reverted back to the non-pretty compiler errors:

https://github.com/go-vela/compiler/pull/133

we just never reverted the ui changes that decode the pretty errors

Reply:
so its more likely that its not needed here, nor the audit/hooks page 😅 
but if we ever try to improve the compiler error messages..... then it will be awesome to have! 
so its up to yall on whether you want to remove it here, remove it in both, leave it, whatever

Reply:
makes sense, hopefully we get there to the pretty errors soon 😅 i'm ok either way, i don't think the overhead is bad, especially for this., since it's already removed, we will go with that, i suppose. thanks @wsan3 @plyr4